Source/Fuse.Controls.Native/DarkMode.uno

```diff
@@ -58,6 +58,10 @@ namespace Fuse.Controls.Native.iOS
 		</App>
 		```
 	*/ 
+	[ForeignInclude(Language.Java,
+		"java.lang.Runnable",
+		"android.content.res.Configuration",
+		"android.app.Activity")]
 	public class DarkMode : NativeEventEmitterModule
 	{
 		public static readonly DarkMode _instance;
```

Comment:
One more `public` that should be dropped, or make `internal` if it's used outside this class.